bench.py

```diff
@@ -483,6 +491,54 @@ def validate_args(args: argparse.Namespace):
                 "with --num-processes 1"
             )
 
+def run_multi_benchmark_with_warmup(
+    num_processes: int,
+    query_processes: int,
+    dataset: str,
+    num_tables: int,
+    batch_size: int,
+    num_queries: int,
+    query_type: str,
+    ingest: bool,
+    index: bool,
+    prefix: str,
+    reset: bool,
+    warmup: bool,
+):
+    if warmup:
+        print("Running warmup benchmark...")
+        run_multi_benchmark(
+            num_processes,
+            query_processes,
+            dataset,
+            num_tables,
+            batch_size,
+            num_queries,
+            query_type,
+            ingest,
+            index,
+            prefix + "-warmup",
```

Comment:
I don't think we need to create and ingest a whole new set of tables

Reply:
Make sense! Change to warmup with query only